addons/website_event/static/src/scss/event_templates_page.scss

```diff
@@ -34,24 +34,23 @@
         }
     }
 
-    .o_wevent_registration_btn {
-        flex: 0 0 100%;
+    .o_wevent_registration_title {
```

Comment:
I don't understand why we give it the "btn" class just to hardcode default cursor and no click behavior behind it.